observability/otel/otel_rn/otel-rn-past-releases.adoc

```diff
@@ -11,13 +11,15 @@ include::modules/otel-product-overview.adoc[leveloffset=+1]
 [id="otel-rn-3-1"]
 == Release notes for {OTELName} 3.1
 
+The {OTELName} is provided through the {OTELOperator}.
+
 [id="otel-rn_3-1_new-features-and-enhancements"]
 === New features and enhancements
 
 //plural: `enhancements:`
 This update introduces the following enhancements:
 
-* {OTELName} 3.1 is based on link:https://opentelemetry.io/[OpenTelemetry] 0.93.0.
+* {OTELName} 3.1 is based on the open source link:https://opentelemetry.io/[OpenTelemetry] release 0.93.0.
```

Comment:
Technically maybe out of scope, but it looks slightly odd to update this for only one release and have different wording for the older releases:
![image](https://github.com/openshift/openshift-docs/assets/40172997/c60abd4b-44a2-403a-b793-51f5efaead71)
Maybe consider a follow up Jira to update the others?

Reply:
:slightly_smiling_face: Thank you for noticing. I previously made the decision to not update the previous releases' release notes because of a number of reasons. I have just updated this line for the previous release notes of OpenTelemetry (which what you're commenting about) but did nothing for the Jaeger and Tempo release notes because the format is different in their earlier releases and I think it's not worth the effort to invest time in extensively redoing past releases' release notes.